DQM/CTPPS/plugins/CTPPSCommonDQMSource.cc

```diff
@@ -143,16 +143,18 @@ void CTPPSCommonDQMSource::GlobalPlots::Init(DQMStore::IBooker &ibooker) {
     ya->SetBinLabel(6, "45, 220, FR-HR");
     ya->SetBinLabel(7, "45, 220, FR-TP");
     ya->SetBinLabel(8, "45, 220, NR-BP");
```

Comment:
I'm just noticing that elsewhere in this PR these are called "NR-BT" (here "NR-BP")...

Reply:
Already replied here: https://github.com/cms-sw/cmssw/pull/41389#discussion_r1184875771